BugReport/Program.cs

```diff
@@ -415,6 +453,71 @@ static ErrorCode Main_Internal(string[] args)
         }
     }
 
+    private static ErrorCode SubscribeIssues(IEnumerable<string> configFilePaths, IEnumerable<string> issueFilePaths, string outputFilePath, string authenticationToken)
+    {
+        var config = new Config(configFilePaths);
+        IEnumerable<DataModelIssue> issues = IssueCollection.LoadIssues(issueFilePaths, config, IssueKindFlags.Issue | IssueKindFlags.PullRequest);
+
+        Repository repo = config.Repositories.First();

```

Comment:
Or rather `Single`?

Reply:
`First` is consistent with the rest of the code, I kept it that way in case there was something I wasn't seeing re: configuration file.